ui/src/app/applications/components/application-details/application-details.tsx

```diff
@@ -281,6 +326,17 @@ export class ApplicationDetails extends React.Component<RouteComponentProps<{nam
                                                     </div>
                                                 )}
                                         </div>
+                                        <SlidingPanel isShown={this.state.showGroupedNodesPanel} onClose={() => this.closeGroupdedNodesPanel()}>
+                                            <ApplicationResourceList
```

Comment:
Have you considered pagination here if there are many resources that are grouped?

Reply:
Yes, there's not an easy way to implement it in a sliding panel, we can have this feature as an improvement in a separate issue as discussed.

Reply:
Sounds good to me as well. Pagination can be added as an ehnancement.